.github/workflows/deploy_rovecomm.yml

```diff
@@ -35,13 +38,35 @@ jobs:
           echo $tag
           git checkout $tag
 
+      - name: Build and Create Executable
+        run: |
+          cd /opt/RoveComm_CPP/
```

Comment:
Remove this `cd` step because the checkout step takes you automatically into that directory.